HDSAviz/Basic_Plot_Updated.py

```diff
@@ -0,0 +1,130 @@
+"""This is a basic script for plotting an analysis file.
+
+    This script is just the first step in visualizing the data set. This file
```

Comment:
It might be useful to let the user's know if there are any packages on which this code might be dependent on like  data_processing.py and probably a brief description of what inputs are being taken from the library.
A note on some new library packages  involved in the code like bokeh that might need to be installed before running the code.might be helpful too.